packages/telemetry/src/otel-and-flight-recorder.js

```diff
@@ -0,0 +1,11 @@
+import { makeSlogSender as makeOtelSlogSender } from './otel-trace.js';
+import { makeSlogSender as makeFlightRecorderSlogSender } from './flight-recorder.js';
+
+export const makeSlogSender = async opts => {
+  const flightRecorderSlogSender = await makeFlightRecorderSlogSender(opts);
+  const otelSlogSender = await makeOtelSlogSender(opts);
+  return obj => {
+    flightRecorderSlogSender(obj);
+    return otelSlogSender(obj);
```

Comment:
Why use `return` here? It seems these senders don't return anything and it makes it feel like we're just ignoring the return value of the flight recorder bit not the otel sender.